com.unity.mobile.notifications/Runtime/Android/AndroidNotification.cs

```diff
@@ -177,11 +177,21 @@ public DateTime CustomTimestamp
             }
         }
 
+        /// <summary>
+        /// Set this notification to be shown when app is in the foreground (default: true).
+        /// </summary>
+        public bool ShowInForeground
+        {
+            get => !m_SilentInForeground;
```

Comment:
Why the inversion ? I mean why not have m_ShowInForeground, or even have a collapsed property?

Reply:
Backward compatibility. This is struct, so if you make one using default, all booleans are false, while this property should default to true, because that was the behavior before it's introduction.

Reply:
This is kind a big inconvenience of struct, what's the reason AndroidNotification is a struct and not a class? Since you might get away this time, but when you'll need a default which is not a "default" of struct, you won't get away with struct.

Reply:
It's been that since long ago.
It actually makes sense, since it's only contains data.
In any case, it is what it is and changing it to class requires better justification than imaginary "what if in future".